Makefile

```diff
@@ -48,11 +48,23 @@ install: check-gopath
 uninstall:
 	rm -f $(BINDIR)/critest
 
+lint:
+	./hack/repo-infra/verify/go-tools/verify-gometalinter.sh
+
+install.tools: .install.gometalinter
+
+.install.gometalinter:
+	go get -u github.com/alecthomas/gometalinter
```

Comment:
For adding other tools in the future. If you think we will not need any other tool, I'm fine with making these two commands under install.tools directly.